### ISSUE TITLE
RMA progress bars are not displaying newly packed product.

### ISSUE THREAD

**Issue by AlexLT1** [commutron/neptune-dev#11]
The bar is being updated, but when hovering over the bar, it will show that new is still "0" packed, but the remaining goes down. Only seems to be happening on rma progress bars, as I tested this on North Star while it wasn't under an rma, and that one had showed that 1 had been newly packed.

**Comment by mattandwhatnot** [comment]
Nothing is broken its just that 'new' progress doesn't get calculated for counters or extended steps. It didn't seem worth the extra complication at the time. Although if the inconsistency causes confusion or doubt then it's more worth it than I thought. 

Important to note that Extension (rma) progress bars are a combination of all the included steps, not just complete. This can sometimes result in a decimal result like 6.5/10. 

To address this I have added "new" calculations as well as specific completed counts to the more information hover. 
Released with v3.5.0 today.
